src/layouts/components/FooterExternalLink.tsx

```diff
@@ -30,21 +27,9 @@ export default function FooterExternalLink(props: {
         className="flex flex-row items-center"
       >
         {t(`footer.sitemap.${category}.${text}`)}
-        <svg width="0" height="0">
-          <linearGradient
-            id="accent-gradient"
```

Comment:
if we remove this, on mouse hover, the up arrow will have no color, this is expected behaviour?

Reply:
oh yes! This was my mistake - will re-include it